hack/gofmt.sh

```diff
@@ -13,7 +13,7 @@ if [ "${IS_CONTAINER}" != "false" ]; then
 
   # shellcheck disable=SC2086
   if [ -n "$(gofmt -l .)" ]; then
-      make fmt
+      gofmt -s -d -e .
```

Comment:
I don't think we need `make fmt` here, because if there is an issue, and when linter Ci is triggered, it will fix the formatting issues too, but it will just happen on the temporary pod, not on the actually patch. So, it is responsibility of the patch author to run `make fmt`.